docs/attribution_parameterization.md

```diff
@@ -0,0 +1,118 @@
+## Attribution Parameterization 
+
+Attributions for different models and use cases can range from simple to more complex. This will help see how to set the various parameters to get what you need.
+
+### Basic Definitions and Terminology
+
+**What is a tensor?**
+A tensor is a multidimensional object that can be model inputs, or layer activations.
```

Comment:
Need to expand on this to make anchors less confusing. Tensor may mean an operation/function that has inputs and has outputs.

Reply:
A tensor is not an operation/function with inputs/outputs, that is a layer. Tensors do not have inputs or outputs. Unless I'm missing something.

Reply:
Ahh, ok. I re-read it. Seems ok.